includes/html/pages/ports.inc.php

```diff
@@ -404,6 +404,13 @@
                     }
                 }
                 break;
+                case 'group':
```

Comment:
This only filters the graph view, and your current code does not work.  Try:
```php
                $where .= ' AND port_id IN (';
                $where .= DB::table('port_group_port')->where('port_group_id', $vars['group'])->pluck('port_id')->implode(',');
                $where .= ')';
```

Reply:
You could also join the table and filter that way, but the query isn't set up for that so it would take a little deeper changes.

To filter the ports table, you'll need to edit app/Http/Controllers/Table/PortsController.php

Reply:
Adding this to the filterFields function
```php
'group' => function ($query, $group) {
    /** @var \Illuminate\Database\Eloquent\Builder $query */
    return $query->whereHas('groups', function ($query) use ($group) {
        /** @var \Illuminate\Database\Eloquent\Builder $query */
        return $query->where('id', $group);
    });
 ```
 and this to ncludes/html/pages/ports/list.inc.php
 ```
 group: '<?php echo $vars['group']; ?>',
 ```